File: translation_output/mistral/codenet/python/java/p03551.java
<s> [INST] Translate the following python code to java and enclose your solution inside ```java```:
```
N,M = list(map(int,input().split()))
time = 1900*M+(N-M)*100
print(time*(2**M))

```
[/INST]
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int M = sc.nextInt();
        int time = 1900 * M + (N - M) * 100;
        System.out.println(time * (1 << M));
    }
}
```</s>

File: translation_output/mistral/codenet/java/python/p03551.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```:
```
import java.util.Scanner;

public class p03551 {

	public static void main(String[] args) {
		p03551 m = new p03551();
		m.run();
	}

	Scanner sc = new Scanner(System.in);

	void run() {

		int n = sc.nextInt();
		int m = sc.nextInt();
		
		
		long base = n*100+m*1800;
		
		System.out.println(base << m);
		
	}
}

```
[/INST]
```python
import sys

class p03551:

    def run(self):
        n = int(input())
        m = int(input())

        base = n*100+m*1800

        print(base << m)

if __name__ == "__main__":
    p03551 = p03551()
    p03551.run()
```</s>
